packages/core/src/lib/markdown-it/plugins/markdown-it-alt-frontmatter.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+// Process "---" => "<frontmatter>
+
+module.exports = function centertext_plugin(md) {
+  function alt_frontmatter(state, startLine, endLine, silent) {
+    var marker, len, nextLine, token,
+      haveEndMarker = false,
+      pos = state.bMarks[startLine] + state.tShift[startLine],
+      max = state.eMarks[startLine];
+    const keyValueRegex = new RegExp("^\\w+:\\s.*");
```

Comment:
```suggestion
    const keyValueRegex = new RegExp("^\\w+:\\s+.*");
```
To accommodate multiple spaces before the value, since our existing frontMatter allows for this